x-pack/packages/kbn-elastic-assistant/impl/assistant/index.tsx

```diff
@@ -409,7 +473,7 @@ const AssistantComponent: React.FC<Props> = ({
                     </EuiFlexItem>
 
                     <EuiFlexItem grow={false}>
-                      <SettingsPopover />
+                      <SettingsPopover isDisabled={currentConversation.replacements == null} />
```

Comment:
If you unhide the lines above, you can see that the other controls in this area are disabled when `currentConversation.replacements == null`. It seemed strange this one would be enabled, so I disabled it for this component as well